### PR TITLE
mention that the private key file is not cached

### PR DESCRIPTION
**Pull request by jfeltesse-mdsol**
Follow up of https://jira.mdsol.com/browse/MCC-655130

Open to suggestions to improve the copy (edit: _of the code comments I added_, this is not some "refactor the README" PR).

@dcassidy-mdsol @mdsol/libraries_dotnet @mdsol/architecture-enablement @mdsol/team-51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,9 +125,9 @@ public async Task<HttpResponseMessage> SignAndSendRequest(HttpRequestMessage req
     }
 }
 ```
-The `SignVersions` parameter can be used to specify which protocol version to sign outgoing requests. Like as:  
-`SignVersions = MAuthVersion.MWS`: signs with `MWS` protocol only.  
-`SignVersions = MAuthVersion.MWS | MAuthVersion.MWSV2` : signs with both `MWS` and `MWSV2` protocol.  
+The `SignVersions` parameter can be used to specify which protocol version to sign outgoing requests. Like as:
+`SignVersions = MAuthVersion.MWS`: signs with `MWS` protocol only.
+`SignVersions = MAuthVersion.MWS | MAuthVersion.MWSV2` : signs with both `MWS` and `MWSV2` protocol.
```
Comment (hkurniawan-mdsol) [inline README.md:130]: Can we have these as bullet points? Or at least in a separate line.
Currently it just one long text.

File: README.md [hunk 2]
```diff
@@ -323,30 +327,30 @@ for authenticating the incoming requests.
 
 ##### What Cryptographic provider is used for the encryption/decryption?
 
-In the latest version of 4.0.0, we are using the available dotnet security [System.Security.Cryptography] which works 
-for both **.NET Framework 4.6.1** and **.NET Standard 2.0** in case of V2 protocol. However, for the continue support 
+In the latest version of 4.0.0, we are using the available dotnet security [System.Security.Cryptography] which works
```
Comment (hkurniawan-mdsol) [inline README.md:330]: Can we put System.Security.Cryptography in a code block?

File: README.md [hunk 3]
```diff
@@ -323,30 +327,30 @@ for authenticating the incoming requests.
...
 
 ##### What are the major changes in the 4.0.0 version?
 
 In this version we have added support for V2 protocol which uses `MCC-Authentication` as MAuthHeader and  `MCC-Time` as
 MAuthTimeHeader. And, this V2 protocol supports for signing and authenticating url with query string parameters.
 For Signing, we added two new options in `MAuthSigningOptions`: `MAuthVersion` which is mandatory and takes enumeration
  value of `MAuthVersion.MWSV2` for V2 protocol or `MAuthVersion.MWS` for continue of using V1 protocol.
-Another option `DisableV1` is `false` by default if not provided. But, it is needed to provide as `true` when the client 
+Another option `DisableV1` is `false` by default if not provided. But, it is needed to provide as `true` when the client
 need to sign on by no more supporting V1 protocol.
```
Comment (hkurniawan-mdsol) [inline README.md:350]: Can we paraphrase this sentence? Kind of hard to grasp.

Comment (jfeltesse-mdsol) [reply README.md:350]: Outside the scope of this PR, the changes here appear because of the trailing whitespace my editor (rightfully).

**Comment by johnduhart** [comment]
This still leaves people who don't fully read the readme every time they get update to get burned. The library needs to be fixed to either properly handle loading the file once, or removing the feature that allows for specifying a path.

File: README.md (hunk 3 above)
Comment (hkurniawan-mdsol) [reply README.md:351]: 😞

**Review by hkurniawan-mdsol — APPROVED** [review]

**Comment by jfeltesse-mdsol** [comment]
anyone else has any comments? going through my open PRs now, seeing this is still open.

**Comment by hkurniawan-mdsol** [comment]
>anyone else has any comments? going through my open PRs now, seeing this is still open.

Nope. Merge and Ship it!

**Review by prajon84 — APPROVED** [review]

**Pull request merged**